### ISSUE TITLE
Can I prevent some react component be accepted?

### ISSUE THREAD

**Issue by zhoudaxia2016** [pmmmwh/react-refresh-webpack-plugin#711]
A situation:
I have a component A whitch extends another component B.
When I edit component B, I want to update component A. 
B is considered as a react refresh boundary, so B accepts updates for itself.

I searched in the source code， I did not find a hook that can check a file if the file is a react refresh boundary myself.
Or I have other way to solve my problem ?

Thanks to view.

**Comment by zhoudaxia2016** [comment]
I have some ideas. Can we pass options to config two things below？
1. registerExportsForReactRefresh. Regist more component when a component update
2. isReactRefreshBoundary. Check if a file is  ReactRefreshBoundary

**Comment by pmmmwh** [comment]
No, this is not supported. React components are supposed to be side-effect free, so they should be fine to reload with full granularity.
